src/gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -19357,14 +19365,14 @@ properties:
       - G2
       - G3
       - G4
-      - GX
       - GB
+      - GX
       - High Grade
       - Intermediate Grade
       - Low Grade
+      - not reported
```

Comment:
Should probably group with the other `Not Reported` or update the documentation why this is belong in the alphabetical section

Reply:
Added fix to script to group lowercase "not reported" with title case "Not Reported"